dcerpc/configure.ac

```diff
@@ -287,12 +286,10 @@ AC_MSG_CHECKING(pthread library and requirements)
 AC_MSG_RESULT()
 
 AC_ARG_WITH(pthread-ext-dir,
-	AC_HELP_STRING([--with-pthread-ext-dir=DIR],
-		       [base directory of pthread extension library]))
+	AS_HELP_STRING([--with-pthread-ext-dir=DIR],[base directory of pthread extension library]))
```

Comment:
Can you also please update the `AC_HELP_STRING` usage in `m4/acinclude.m4`? That still produces warnings.